<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Fasthenry 3.0wr Windows GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CheckVersionButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run docker -v to check version or if docker is installed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Check docker version</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OutputBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output of execuation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartDockerButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start docker desktop for windows. Need to install it first.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Start docker desktop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BuildImageButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Build image from Dockerfile</string>
    </property>
    <property name="whatsThis">
     <string>First generate Dockerfile then generate the image.</string>
    </property>
    <property name="text">
     <string>Build Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Tips: &lt;br&gt;1. Move mouse over buttons to see instructions.&lt;br&gt;2. check the input commands and their outputs.&lt;br&gt;3. put the fasthenry file, Dockerfile and .inp file in the same folder of this program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunFasthenry">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Build image from Dockerfile</string>
    </property>
    <property name="whatsThis">
     <string>First generate Dockerfile then generate the image.</string>
    </property>
    <property name="text">
     <string>Run fasthenry</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
